packages/backend/src/auth.ts

```diff
@@ -40,9 +40,10 @@ export default () => {
         '/:ensName',
         async (req: express.Request & { app: WithLocals }, res, next) => {
             try {
-                const ensName = Lib.account.normalizeEnsName(
+                const ensName = await req.app.locals.db.getId(
```

Comment:
maybe rename ensName to id or something since the variable is no longer containing an ens name

Reply:
I'll change the variable to `IdEnsName` and the function to `getIdEnsName` since it still contains an ENS name.
The `getIdEnsName` function returns the initial `0x123....addr.dm3.eth` ENS name which is internally used as the ID.